runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsConfig.java

```diff
@@ -64,6 +64,17 @@ default boolean clue()
 		return true;
 	}
 
+	@ConfigItem(
+			position = 3,
+			keyName = "killcount",
+			name = "Killcount Command",
+			description = "Configures whether the Killcount command is enabled"
+	)
+	default boolean killcount()
+	{
+		return true;
+	}
+
 	@ConfigItem(
 		position = 3,
```

Comment:
You shouldn't have multiple config items with the same position--this must be incremented.

Reply:
My bad, forgot to change that.